StylisticConfigArray.mjs

```diff
@@ -0,0 +1,356 @@
+/* eslint-disable max-lines, no-magic-numbers */
+
+import StylisticPlugin from "@stylistic/eslint-plugin";
+
+
+const StylisticConfigArray = [
+    StylisticPlugin.configs["disable-legacy"],
+    {
+        plugins: {
+            "@stylistic": StylisticPlugin,
+        },
+        rules: {
+            "@stylistic/array-bracket-newline": [
+                "error",
+                "consistent",
+            ],
+            "@stylistic/array-bracket-spacing": [
+                "error",
+                "never",
+            ],
+            "@stylistic/array-element-newline": [
+                "error",
+                {ArrayExpression: "always", ArrayPattern: {minItems: 3}},
+            ],
+            "@stylistic/arrow-parens": ["error"],
+            "@stylistic/arrow-spacing": ["error"],
+            "@stylistic/block-spacing": [
+                "error",
+                "never",
```

Comment:
Shouldn't this be "always"?

Reply:
This conflicts with https://eslint.style/rules/default/object-curly-spacing

Reply:
Right, I think we previously discussed and settled with no spaces in blocks because the default settings of many rules prefer that principle. However, this `@stylistic/block-spacing: "always"` default has been a first exception.

I agree in single-line objects, spaces will improve clarity. Can we create an issue and defer it to another PR since there could be multiple rules we need to configure to ensure consistency? e.g., import-related rules, curly-related rules, block-related rules